unity/Assets/Maroon/scenes/experiments/FallingBallViscosimeter/Scripts/ViscosimeterManager.cs

```diff
@@ -1,185 +1,168 @@
 using System;
-using System.Collections;
 using System.Collections.Generic;
+using GameLabGraz.UI;
+using GEAR.Localization.Text;
 using Maroon.scenes.experiments.FallingBallViscosimeter.Scripts;
-using PlatformControls.PC;
+using Maroon.UI;
 using UnityEngine;
 using TMPro;
 using Object = UnityEngine.Object;
```

Comment:
I think you can remove this if you apply the suggestion below for the `GetAllMeasurableObjects` method